macros/object_pivot.sql

```diff
@@ -12,7 +12,11 @@
 
```

Comment:
this is the basic idea but I need to touch base with @jennaallen on how to test locally

Reply:
I figured out how to test locally. This now generates the code: 

```-- depends_on: emilyekdahl.stg_policy_events__location_specific_rater_inputs_flattened
select policy_id, event_id,location_id
    from emilyekdahl.stg_policy_events__location_specific_rater_inputs_flattened
```

Reply:
did you run that test?

Reply:
yes i did, why did it not work for you?